en-US/win10/StartCoding.md

```diff
@@ -336,6 +336,15 @@ lang: en-US
                     <a href="{{site.baseurl}}/{{page.lang}}/win10/samples/WeatherStationLightning.htm">Weather Station (C#/C++)</a><br>
                     <a href="{{site.baseurl}}/{{page.lang}}/win10/samples/SPIDisplayLightning.htm">SPI Display (C#/C++)</a>
                   </div>
+				  
+                </div>
+
+                <div class="row section-heading">
+                    <div class="col-md-3">
+                        <h4>Controlling DC Motors</h4>
+                        <p>Learn to control DC motors using a Windows IoT Core enabled device!</p>
+                        <a href="{{site.baseurl}}/{{page.lang}}/win10/samples/arduino-wiring/DCMotor.htm">DC Motors (Wiring)</a>
```

Comment:
not currently (to my knowledge) but this is how we set these links currently. It will guarantee we don't have to modify the links if we provide the page in another language. The link will automatically grab the 'lang' property from the page to forward them to the linked page of the same language